src/examples/USEnrichmentExample.cs

```diff
@@ -0,0 +1,87 @@
+namespace Examples
+{
+	using System;
+	using System.Net;
+	using System.Collections.Generic;
+	using System.IO;
+    using System.Linq;
+    using SmartyStreets;
+	using SmartyStreets.USEnrichmentApi;
+    using System.Reflection;
+
+	internal static class USEnrichmentExample
+	{
+		public static void Run()
+		{
+            // specifies the TLS protocoll to use - this is TLS 1.2
+            const SecurityProtocolType tlsProtocol1_2 = (SecurityProtocolType)3072;
+
+            // var authId = "Your SmartyStreets Auth ID here";
+            // var authToken = "Your SmartyStreets Auth Token here";
+
+            // We recommend storing your keys in environment variables instead---it's safer!
+            var authId = Environment.GetEnvironmentVariable("SMARTY_AUTH_ID");
+			var authToken = Environment.GetEnvironmentVariable("SMARTY_AUTH_TOKEN");
+			ServicePointManager.SecurityProtocol = tlsProtocol1_2;
+
+			var client = new ClientBuilder(authId, authToken).BuildUsEnrichmentApiClient();
```

Comment:
client is an IDisposable and needs to be cleaned up because it contains resources outside of the garbage collector. This can be mitigated with the keyword "using" before the keyword "var" so that it cleans up when if falls out of scope.

Reply:
Ignore this comment. I looked deeper into it. It looks like the default NativeSender uses the HttpWebRequest class which is not an IDisposable. But the end user CAN specifier an ISender which uses the HttpClient class. But in those cases it is up to the end user to take ownership of their memory management. So this is fine.

Reply:
Love all this insight!